### PR TITLE
Create event card in community page

### PR DESCRIPTION
**Pull request by bfgonzalez**
* initial commit w/ placeholder text and image
* no states have been applied to the SectionToggle yet (i.e. for switching between "Upcoming" and "Past")

### CODE REVIEW

File: src/components/pages/Community/Community.js [hunk 1]
```diff
@@ -17,6 +24,36 @@ class Community extends Component {
...
+          <SectionToggle
+            toggleLabel1="Upcoming"
+            toggleLabel2="Past"/>
+          <div className="event-card">
+            <div className="event-info">
+              <p className="event-label">Information Architecture</p>
+              <h2 className="event-name">UX&Chill Ep 08</h2>
+
+              <ul className="event-details">
+                <img src={calendar} alt="calender"/><li>24 September 2018, 5:00 to 7:30 PM</li>
```
Comment (xaviablaza) [inline src/components/pages/Community/Community.js]: This should be temporarily be a prop. We may need to discuss how we will record event start and end times in the database.

File: src/components/pages/Community/Community.js [hunk 2]
```diff
@@ -17,6 +24,36 @@ class Community extends Component {
...
+        <div className="community-events-container">
+          <p className="section-label">Events</p>
+          <h2>What's happening</h2>
+          <SectionToggle
+            toggleLabel1="Upcoming"
+            toggleLabel2="Past"/>
+          <div className="event-card">
+            <div className="event-info">
+              <p className="event-label">Information Architecture</p>
+              <h2 className="event-name">UX&Chill Ep 08</h2>
```
Comment (xaviablaza) [inline src/components/pages/Community/Community.js]: Pass the `event-label` and `event-name` as props.

File: src/components/pages/Community/Community.js [hunk 3]
```diff
@@ -17,6 +24,36 @@ class Community extends Component {
...
+
+              <ul className="event-details">
+                <img src={calendar} alt="calender"/><li>24 September 2018, 5:00 to 7:30 PM</li>
+                <img src={location} alt="location"/><li>Faura Hall, Ateneo de Manila University</li>
+                <img src={speaker} alt="speaker"/><li>Avery Si</li>
+              </ul>
```
Comment (xaviablaza) [inline src/components/pages/Community/Community.js]: `location` and `speaker` should also be props.

File: src/components/pages/Community/Community.js [hunk 4]
```diff
@@ -17,6 +24,36 @@ class Community extends Component {
...
+
+              <h4 className="about-header">About the event</h4>
+              <p className="about-desc">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+              <h4 className="about-header">About the speaker</h4>
+              <p className="about-desc">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
```
Comment (xaviablaza) [inline src/components/pages/Community/Community.js]: `about-event` and `about-speaker` should be passed in as props.

File: src/components/pages/Community/Community.js [hunk 5]
```diff
@@ -17,6 +24,36 @@ class Community extends Component {
...
+
+              <div className="register-btn">
+                <a className="register" href="/">Register Now</a>
+                <img className="register-icon" src={registerArrow} alt="arrow"/>
+              </div>
+            </div>
+            <img className="event-img" src="https://images.unsplash.com/photo-1538981177685-1853528f4ab7?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=e4636f39faac7e08212d191ecb3ab05d&auto=format&fit=crop&w=1569&q=80" alt="event"/>
```
Comment (xaviablaza) [inline src/components/pages/Community/Community.js]: Event image should be passed in as a prop. We should make a guideline for how images will be cropped.

File: src/components/pages/Community/Community.js [hunk 6]
```diff
@@ -17,6 +24,36 @@ class Community extends Component {
...
+                <img src={speaker} alt="speaker"/><li>Avery Si</li>
+              </ul>
+
+              <h4 className="about-header">About the event</h4>
+              <p className="about-desc">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+              <h4 className="about-header">About the speaker</h4>
+              <p className="about-desc">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+
+              <div className="register-btn">
+                <a className="register" href="/">Register Now</a>
```
Comment (xaviablaza) [inline src/components/pages/Community/Community.js]: The `href` here should be a prop.

**Comment by bfgonzalez** [comment]
@xaviablaza pushed new commits to the branch which should resolve the issues you raised. Thanks!

**Comment by xaviablaza** [comment]
Checking now.

File: src/components/pages/Community/Community.js [hunk 7]
```diff
@@ -1,18 +1,17 @@
...
 import registerArrow from './../../../resources/img/register-arrow.svg';
-import calendar from './../../../resources/img/calendar.svg';
-import location from './../../../resources/img/location.svg';
-import speaker from './../../../resources/img/speaker.svg';
 
 import './style.css';
 
 class Community extends Component {
   render() {
+    const { registerLink } = this.props;
```
Comment (xaviablaza) [inline src/components/pages/Community/Community.js]: The `registerLink` should be its own component like you did for the others.

File: src/components/pages/Community/Community.js [hunk 8]
```diff
@@ -32,26 +31,25 @@ class Community extends Component {
...
-              <h4 className="about-header">About the event</h4>
-              <p className="about-desc">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
-              <h4 className="about-header">About the speaker</h4>
-              <p className="about-desc">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+              <AboutHeader aboutHeader="About the event"/>
+              <AboutDesc aboutDesc="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua."/>
+              <AboutHeader aboutHeader="About the speaker"/>
+              <AboutDesc aboutDesc="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua."/>
 
               <div className="register-btn">
```
Comment (xaviablaza) [inline src/components/pages/Community/Community.js]: Put the `register-btn` div in another component with a prop for the href.

**Comment by bfgonzalez** [comment]
@xaviablaza pushed new commits to the branch (re: componentized the _Register Now_ button) which should make the button reusable throughout the Community Page. Thanks!

**Pull request merged**